test/test_transforms_v2_refactored.py

```diff
@@ -3009,3 +3013,104 @@ def test_auto_augment_policy_error(self):
     def test_aug_mix_severity_error(self, severity):
         with pytest.raises(ValueError, match="severity must be between"):
             transforms.AugMix(severity=severity)
+
+
+class TestConvertBoundingBoxFormat:
+    old_new_formats = pytest.mark.parametrize(
+        ("old_format", "new_format"), list(itertools.product(iter(tv_tensors.BoundingBoxFormat), repeat=2))
+    )
+
+    @old_new_formats
+    @pytest.mark.parametrize("dtype", [torch.int64, torch.float32])
+    @pytest.mark.parametrize("device", cpu_and_cuda())
+    def test_kernel(self, old_format, new_format, dtype, device):
+        check_kernel(
+            F.convert_bounding_box_format,
+            make_bounding_boxes(format=old_format, dtype=dtype, device=device),
+            new_format=new_format,
+            old_format=old_format,
+        )
+
+    @old_new_formats
+    @pytest.mark.parametrize("dtype", [torch.float32, torch.uint8])
+    @pytest.mark.parametrize("device", cpu_and_cuda())
+    def test_kernel_inplace(self, old_format, new_format, dtype, device):
+        input = make_bounding_boxes(format=old_format, dtype=dtype, device=device).as_subclass(torch.Tensor)
+        input_version = input._version
+
+        if old_format == new_format:
```

Comment:
What is the expected behaviour when `inplace` is False?

Reply:
Might be worth asserting the behaviour with `inplace=False` as well then? It's a bit strange to omit it: it looks like it's on purpose